app/views/applications_mailer/member_access.html.haml

```diff
@@ -5,13 +5,11 @@
     %p
       Hi, #{@user.name}!
     %p
-      Thanks for getting all of your Double Union membership things set up. You now have access to the
-      #{ link_to "members Google Drive folder", "https://drive.google.com/#folders/0B6a_aDP-2fOVV2FQLW5FVTZ2Mjg" } and a Google calendar called "DU Members". You should read through the
-      #{ link_to "Member Handbook", "https://docs.google.com/document/d/1yYXAj8rzQMiYt2xFdzm2xEOe0LCHohWNrMWzad-4mtQ/edit" } now to get a quick overview of how things work at DU.  You also now have access to the { link_to "DU chat system (Slack)", "https://doubleunion.slack.com/" }.
+      Thanks for getting all of your Double Union membership things set up. You now have access to the mailing list, the #{ link_to "members Google Drive folder", "https://drive.google.com/#folders/0B6a_aDP-2fOVV2FQLW5FVTZ2Mjg" } and a Google calendar called "Double Union Members". You should read through the #{ link_to "Member Handbook", "https://docs.google.com/document/d/1yYXAj8rzQMiYt2xFdzm2xEOe0LCHohWNrMWzad-4mtQ/edit" } now to get a quick overview of how things work at DU.  You also now have access to the { link_to "DU chat system (Slack)", "https://doubleunion.slack.com/" }!
```

Comment:
Whoops, I forgot to double check this email before merging it in 😬 

`{ link_to "DU chat system (Slack)", "https://doubleunion.slack.com/" }` doesn't interpolate in haml -- you need the leading `#`. I'll fix it in a follow up commit.